package.json

```diff
@@ -139,6 +141,23 @@
     "test:php": "phpunit",
     "test:php:help": "npm run test:php -- --help"
   },
+  "lint-staged": {
+    "package.json": [
+      "npm run lint:pkg-json"
+    ],
+    "**/*.css": [
+      "npm run lint:css:fix"
+    ],
+    "**/*.js": [
+      "npm run lint:js:fix"
```

Comment:
Is this "fixing" linting issues here? If so, wouldn't that be incompatible with "staged changes only", as the changes it would make would then not be staged. So you fix something but won't commit that fix, unless you remember to use `git add` again.

Reply:
Yes that makes sense. What's more is that when the `--fix` option is specified when linting JS or CSS files, all the eligible files are linted, instead of the staged ones. I'm not sure if it's a linter configuration error or bug with the linter itself.

I've removed the "fixing" of the lint issues in https://github.com/ampproject/amp-wp/pull/4441/commits/76a97167473966a7311c92343fde871436c0434e.